### ISSUE TITLE
Chern vehicles - A3 Armory

### ISSUE THREAD

**Issue by xlucky13z** [A3Armory/A3A_A3Wasteland_Sock.Altis#51]
I notice whenever I am in the northern part of Chenarous that I have to run everywhere. Vehicles don't seem to spawn at the northern gun store and general store areas. As well as the surrounding towns. 

 Is there any way we could increase the spawn of vehicles in this area. Maybe just regenerating quads at least?

**Comment by codexGW** [comment]
I'll check it out next time I'm on Chernarus. I'm not sure how much time its worth because when the new map comes out it looks like Chernarus might go down.

**Comment by xlucky13z** [comment]
Yeah I can understand it's not becoming very popular. Which is to bad. It's also having a save issue.

 Just thought it was an easy fix to add random vehicles to an area.
